### ISSUE TITLE
re-run diagnostic script stuck

### ISSUE THREAD

**Issue by tomaslovato** [ESMValGroup/ESMValCore#136]
After the successful execution of a recipe,  re-running the diagnostic script stop with the following message:

```
Script will abort to prevent accidentally overwriting your data in these directories
Use -f or --force to force emptying the output directories or use -i or --ignore-existing to ignore existing output directories.
Script will abort to prevent accidentally overwriting your data in these directories
```

This can be easily solved by adding  the suggested options `-f` or `-i`, but I fell like that the re-run message produced by code at the end of diagnostic function should more clear, e.g. by including at least `-i`.

In particular, `-i` can be added in `rerun_msg` variable within the following piece of code:
https://github.com/ESMValGroup/ESMValCore/blob/82178a9c593ce063baa10e8e883c9ba58491cb61/esmvalcore/_task.py#L409-L414
It could be also an idea to add in the log informations the need to specify `-f` or `-i` options.

**Comment by bouweandela** [comment]
Note that those options are specific for Python diagnostics, they are not implemented in NCL, R, or Julia.

**Comment by tomaslovato** [comment]
@bouweandela I recognize it is rather trivial, although not immediate, so I'll close this issue if you agree...

**Comment by bouweandela** [comment]
It's up to you, I think it's a good suggestion for improving the user friendliness of the tool. Unfortunately we have limited time available to implement all of those suggestions...

**Comment by tomaslovato** [comment]
@bouweandela I report here below a tiny change to the above referred code to incldue the `-i` options in the log info when using python diags ... 

``` python
@@ -411,6 +411,8 @@ class DiagnosticTask(BaseTask):
             rerun_msg += ' '.join('{}="{}"'.format(k, env[k]) for k in env
                                   if k not in os.environ)
         rerun_msg += ' ' + ' '.join(cmd)
+        if cmd[1].endswith('.py'):
+            rerun_msg = rerun_msg.replace('.py', '.py -i', 1)
         logger.info("To re-run this diagnostic script, run:\n%s", rerun_msg)

         try:
```
If it looks ok, should I open a PR or you want to add it in the code ....

**Comment by tomaslovato** [comment]
This was solved via #326
